--- ./Makefile.in.org	Thu May  9 20:58:25 2002
+++ ./Makefile.in	Fri May 31 11:41:58 2002
@@ -8,7 +8,10 @@
 SLIBS=
 PROGRAM= lsat
 INSTALL = /usr/bin/install -c
-prefix = /usr/local
+prefix = @prefix@
+BINDIR = @bindir@
+DATADIR = @datadir@
+MANDIR = @mandir@
 
 OBJECTS= $(SOURCES.c:.c=.o)
 
@@ -19,7 +22,8 @@
 all debug: $(PROGRAM)
 
 install:
-	$(INSTALL) $(PROGRAM) $(prefix)/bin
+	$(INSTALL) -d $(DESTDIR)$(BINDIR)
+	$(INSTALL) $(PROGRAM) $(DESTDIR)$(BINDIR)
 
 $(PROGRAM): $(INCLUDES) $(OBJECTS) 
 	$(LINK.c) -o $@ $(OBJECTS) $(SLIBS)
@@ -33,7 +37,8 @@
 sed 's/Perl/LSAT/g'\
 > $(PROGRAM).1
 	rm -f  $(PROGRAM).tmp
-	cp -vf $(PROGRAM).1 debian/manpage.1.ex	
+#	cp -vf $(PROGRAM).1 debian/manpage.1.ex	
 
 installman:
-	cp $(PROGRAM).1 $(prefix)/man/man1/
+	$(INSTALL) -d $(DESTDIR)$(MANDIR)/man1
+	$(INSTALL) $(PROGRAM).1 $(DESTDIR)$(MANDIR)/man1/
